packages/scanner/src/page.ts

```diff
@@ -41,6 +41,10 @@ export class Page {
             return { unscannable: true, error: `Cannot scan ${url} because it is not a html page.` };
         }
```

Comment:
can you also take care of the exceptions that will be thrown by the goto url promise?
https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md#pagegotourl-options

Reply:
Any exceptions thrown in scanning are caught by the Scanner class and passed on. Since we're now passing all error messages back to the user, exceptions from goto will already be returned to the user as well.

Reply:
so, we already have corresponding error codes for the exceptions returned by gotopromise?

Reply:
Added new error codes just now for the goto errors